docs/markdown/Introduction/media.md

```diff
@@ -298,21 +293,39 @@ May 13, 2020
 
 ### Pants Build project's official channel
 
-Home: <https://www.youtube.com/@pantsbuild>  
+Home: <https://www.youtube.com/channel/UCCcfCbDqtqlCkFEuENsHlbQ>  
 Pants Build 2 Tour: <https://www.youtube.com/playlist?list=PLwPRXwjURiOzXjgqydxZE9RVjZqaB6OXb>
 
 ## Repositories
 
+### liga-pytorch
+
+Makes open source project [`rigai`](https://github.com/eto-ai/rikai) more modular and extensible by replacing setuptools with Pants.
```

Comment:
rikai is separated in three projects:
+ https://github.com/liga-ai/liga
+ https://github.com/liga-ai/ligavision
+ https://github.com/liga-ai/liga-pytorch